<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonForm</class>
 <widget class="QDialog" name="PersonForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>&amp;Name</string>
       </property>
       <property name="buddy">
        <cstring>nameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="nameEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="emailLabel">
       <property name="text">
        <string>&amp;Email</string>
       </property>
       <property name="buddy">
        <cstring>emailEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="emailEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="birthdayLabel">
       <property name="text">
        <string>&amp;Birthday</string>
       </property>
       <property name="buddy">
        <cstring>birthdayEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QDateEdit" name="birthdayEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="balanceLabel">
       <property name="text">
        <string>&amp;Balance</string>
       </property>
       <property name="buddy">
        <cstring>balanceEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="ageLabel">
       <property name="text">
        <string>&amp;Age</string>
       </property>
       <property name="buddy">
        <cstring>ageEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="ageEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="resumeLabel">
       <property name="text">
        <string>&amp;Resum√©</string>
       </property>
       <property name="buddy">
        <cstring>resumeDownloadButton</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="resumeDownloadButton">
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="resumeUploadButton">
       <property name="text">
        <string>Upload</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="balanceEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PersonForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PersonForm</receiver>
   <slot>updatePersonData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PersonForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updatePersonData()</slot>
 </slots>
</ui>
